Cython/Utility/CpdefEnums.pyx

```diff
@@ -51,12 +51,14 @@ if PY_VERSION_HEX >= 0x03040000:
         ('{{item}}', {{item}}),
         {{endfor}}
     ]))
+    {{name}}.__doc__ = """{{enum_doc}}"""
+
     {{for item in items}}
     __Pyx_globals['{{item}}'] = {{name}}.{{item}}
     {{endfor}}
 else:
     class {{name}}(__Pyx_EnumBase):
-        pass
+        __doc__ = """{{enum_doc}}"""
```

Comment:
Wouldn't `repr()` work here?
```suggestion
        {{ repr(enum_doc) }}
```

Reply:
Right, I wasn't familiar with the `repr` specific before. I updated the other half of the `if` to also use `repr`.